### ISSUE TITLE
Test failure: config.reload.automatic

### ISSUE THREAD

**Issue by jakelandis** [elastic/logstash#8347]
The following test will error occasionally:

```
19:31:25 Failures:
19:31:25 
19:31:25   1) LogStash::Agent Agent execute options when `config.reload.automatic` is set to`FALSE` and successfully load the config converge only once
19:31:25      Failure/Error: expect(source_loader.fetch_count).to eq(1)
19:31:25 
19:31:25        expected: 1
19:31:25             got: 0
19:31:25 
19:31:25        (compared using ==)
19:31:25      # /var/lib/jenkins/workspace/elastic+logstash+master+multijob-unix-compatibility/os/opensuse/logstash-core/spec/logstash/agent/converge_spec.rb:108:in `block in (root)'
19:31:25      # /var/lib/jenkins/workspace/elastic+logstash+master+multijob-unix-compatibility/os/opensuse/vendor/bundle/jruby/2.3.0/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:46:in `block in /var/lib/jenkins/workspace/elastic+logstash+master+multijob-unix-compatibility/os/opensuse/vendor/bundle/jruby/2.3.0/gems/rspec-wait-0.0.9/lib/rspec/wait.rb'
19:31:25      # /var/lib/jenkins/workspace/elastic+logstash+master+multijob-unix-compatibility/os/opensuse/lib/bootstrap/rspec.rb:13:in `<main>'
19:31:25 
19:31:25 Finished in 7 minutes 45 seconds (files took 19.11 seconds to load)
19:31:25 2688 examples, 1 failure, 3 pending
19:31:25 
19:31:25 Failed examples:
19:31:25 
19:31:25 rspec /var/lib/jenkins/workspace/elastic+logstash+master+multijob-unix-compatibility/os/opensuse/logstash-core/spec/logstash/agent/converge_spec.rb:105 # LogStash::Agent Agent execute options when `config.reload.automatic` is set to`FALSE` and successfully load the config converge only once
19:31:25 
19:31:25 Randomized with seed 8762
```

**Comment by jakelandis** [comment]
closing as duplicate of https://github.com/elastic/logstash/issues/8204
